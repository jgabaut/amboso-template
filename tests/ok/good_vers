#!/bin/bash
"./anvil" -D "./amboso-tags" -q 0.2.4

