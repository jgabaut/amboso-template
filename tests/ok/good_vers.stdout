$
^[[1;33m[QUERY]    ( 0.2.4 ) binary not found in ( ./amboso-tagsv0.2.4 ).^[[0m$
